integration_test/testutil_test.go

```diff
@@ -166,7 +165,7 @@ func (it *ITest) WithIndex() *ITest {
 // WithEnv sets an environment variable for the krew run.
 func (it *ITest) WithEnv(key string, value interface{}) *ITest {
 	if key == "KREW_ROOT" {
-		glog.V(1).Infoln("Overriding KREW_ROOT in tests is forbidden")
+		it.t.Log("Overriding KREW_ROOT in tests is forbidden")
```

Comment:
out of scope but I think this is more suitable for a `t.Fatal`

Reply:
It's a small but important thing, so I changed that.